lib/modis/persistence.rb

```diff
@@ -185,7 +185,7 @@ def create_or_update(args = {})
       validate(args)
       future = persist
 
-      if future && (future == :unchanged || future.value == 'OK')
+      if future && ((future.is_a?(Symbol) && future == :unchanged) || future.value == 'OK')
```

Comment:
Could you explain this change a bit more?

Reply:
Of course! This is to fix a deprecation warning when using `modis` with the latest version of the `redis` gem, 4.2.1:

```
The methods == and != are deprecated for Redis::Future and will be removed in 4.2.0 - You probably meant to call .value == or .value != (.../modis/lib/modis/persistence.rb:188:in `create_or_update')
```

You can reproduce this deprecation warning as follows, working off of the current release of `modis`:
- Run `bundle update redis`
- Run `bundle exec rspec`

I realize this particular solution looks a little odd. It seemed like the most minimal change without trying to refactor how this code uses the `:unchanged` symbol, but maybe more refactoring would be worthwhile? I'm not sure.

Reply:
> The existing code allows `future` to be _either_ a `Redis::Future` instance or a `Symbol`

I see, I was wondering where the `Symbol` is coming from. Thanks for the explanation!